content/about/faq.md

```diff
@@ -68,121 +68,164 @@ Here are some Frequently Asked Questions about Haiku. For development related to
-Yes! You'll find a package for Caya, the multi-protocol IM client. Several alternatives are also available.
+Yes! There are actually multiple messaging applications available for Haiku for various protocols, such as IRC (Vision, Quassel, WeeChat, irrsi),
+Matrix (Quaternion, nheko) and XMPP (Renga, Caya, Vacuum).
+
+**Vision**, **Renga** and **Caya** were speciically developed for the Haiku operating system. They are being maintained by members of our community.
 
 ### Is there Java support?
 Yes! The OpenJDK virtual machine is available and enables you to run applications such as Netbeans, JDownloader, and much more.
 
 ### Is there Flash support?
-Not at the moment. Flash is proprietary technology from Adobe, making it difficult to support for a small open-source operating system such as Haiku. However, there are ways to enjoy the web without Flash, such as using HTML5 compatible versions of webpages (youtube and grooveshark support this for example). Besides, Adobe [deprecated Flash](https://theblog.adobe.com/adobe-flash-update/) anyway.
+Not at the moment. Flash is proprietary technology from Adobe, making it difficult to support for a small open-source operating system such as Haiku.
+However, there are ways to enjoy the web without Flash, such as using HTML5 compatible versions of webpages (youtube and grooveshark support this for
+example). Besides, Adobe [deprecated Flash](https://theblog.adobe.com/adobe-flash-update/) anyway.
 
 ### Is there an office suite?
 There are several options:
 
   * You can install [LibreOffice](https://www.libreoffice.org/) available at _HaikuDepot_.
   * You can find the [Caligra](https://www.calligra.org/) office suite at _HaikuDepot_.
-  * You can use an online tool such as [Google Docs](https://docs.google.com).
+  * You can use online tools, such as [Google Docs](https://docs.google.com).
   * You can use native applications, such as Sum-It (spreadsheet), WonderBrush (drawing and graphical design).
-  * You can run GoBe productive, the office suite for BeOS, if you have a copy of it. Unfortunately, it is not distributed anymore.
+  * If you happen to own an old copy of GoBe productive laying around, you can use it on Haiku with the 32-bit version of Haiku.
+    Unfortunately, it is not being distributed anymore.
 
 ### Are there any games for Haiku?
-Yes! _HaikuDepot_ offers a selection of open source and freeware games. Most games written for BeOS will also run fine, and you can use emulators such as DOSBox to run games originally written for other systems as well.
+Yes! _HaikuDepot_ offers a selection of open source and freeware games. Most games written for BeOS will also run fine,
+and you can use emulators, such as DOSBox, Dolphin, snes9x MAME and RetroArch, to run games originally written for
+other systems and consoles as well.
 
-### Is there python, ruby, perl?
+### Are Python, Ruby, Perl and Go available on Haiku?
 Yes! Haiku includes these common scripting languages and many more.
```

Comment:
That is misleading for Go (not currently in our package manager). You could mention Rust instead.